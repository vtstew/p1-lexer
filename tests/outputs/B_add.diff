--- outputs/B_add.txt	2022-09-02 21:52:02.389703085 -0400
+++ expected/B_add.txt	2022-08-19 09:55:09.000000000 -0400
@@ -0,0 +1,19 @@
+KEYWORD  [line 001]  def
+KEYWORD  [line 001]  int
+ID       [line 001]  main
+SYMBOL   [line 001]  (
+SYMBOL   [line 001]  )
+SYMBOL   [line 002]  {
+KEYWORD  [line 003]  int
+ID       [line 003]  a
+SYMBOL   [line 003]  ;
+ID       [line 004]  a
+SYMBOL   [line 004]  =
+DECLIT   [line 004]  4
+SYMBOL   [line 004]  +
+DECLIT   [line 004]  5
+SYMBOL   [line 004]  ;
+KEYWORD  [line 005]  return
+ID       [line 005]  a
+SYMBOL   [line 005]  ;
+SYMBOL   [line 006]  }
