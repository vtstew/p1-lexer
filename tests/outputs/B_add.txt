KEYWORD  [line 001]  def
KEYWORD  [line 001]  int
ID       [line 001]  main
SYMBOL   [line 001]  (
SYMBOL   [line 001]  )
SYMBOL   [line 001]  {
SYMBOL   [line 001]  	int a;
SYMBOL   [line 001]  	a = 4 + 5;
SYMBOL   [line 001]  	return a;
SYMBOL   [line 001]  }
