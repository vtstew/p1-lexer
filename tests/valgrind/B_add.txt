==3158076== Memcheck, a memory error detector
==3158076== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3158076== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3158076== Command: ../decaf inputs/add.decaf
==3158076== 
Invalid token!
==3158076== 
==3158076== HEAP SUMMARY:
==3158076==     in use at exit: 35,252 bytes in 231 blocks
==3158076==   total heap usage: 336 allocs, 105 frees, 51,932 bytes allocated
==3158076== 
==3158076== LEAK SUMMARY:
==3158076==    definitely lost: 272 bytes in 5 blocks
==3158076==    indirectly lost: 34,980 bytes in 226 blocks
==3158076==      possibly lost: 0 bytes in 0 blocks
==3158076==    still reachable: 0 bytes in 0 blocks
==3158076==         suppressed: 0 bytes in 0 blocks
==3158076== Rerun with --leak-check=full to see details of leaked memory
==3158076== 
==3158076== For lists of detected and suppressed errors, rerun with: -s
==3158076== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
