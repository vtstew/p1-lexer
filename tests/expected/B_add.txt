KEYWORD  [line 001]  def
KEYWORD  [line 001]  int
ID       [line 001]  main
SYMBOL   [line 001]  (
SYMBOL   [line 001]  )
SYMBOL   [line 002]  {
KEYWORD  [line 003]  int
ID       [line 003]  a
SYMBOL   [line 003]  ;
ID       [line 004]  a
SYMBOL   [line 004]  =
DECLIT   [line 004]  4
SYMBOL   [line 004]  +
DECLIT   [line 004]  5
SYMBOL   [line 004]  ;
KEYWORD  [line 005]  return
ID       [line 005]  a
SYMBOL   [line 005]  ;
SYMBOL   [line 006]  }
